### ISSUE TITLE
There are a few required updates for the main editor since the newest CEF release

### ISSUE THREAD

**Issue by smithed** [kalyanramu/SPIFramework#3]
Come see us for the changes. Unfortunately since you were using a pre-release version the code looks to be broken right now. I'd guess it should only take about 30 minutes to get up and running again.

**Comment by smithed** [comment]
Also it looks like there was a treecontrol VI which got modified in your copy which breaks it on my machine

**Comment by ianwknox** [comment]
Yeah, I modified the AddNode.vi slightly to return the current node's reference so I could activate the node that is being added to the tree

**Comment by smithed** [comment]
Instead you can just index the summary array to get the node ID.
